go.mod

```diff
@@ -72,37 +72,38 @@ require (
 	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
 	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/cobra v1.0.0
-	github.com/stoewer/go-strcase v1.1.0 // indirect
 	github.com/stretchr/testify v1.6.1
 	github.com/terraform-provider-openstack/terraform-provider-openstack v1.32.0
-	github.com/terraform-providers/terraform-provider-aws v0.0.0
-	github.com/terraform-providers/terraform-provider-azurerm v0.0.0
+	github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20200807230610-d5346d47e3af
```

Comment:
is the change for aws and azure intentional?

Reply:
No, it was automatically changed during `go mod tidy`